ofctrl/ofSwitch.go

```diff
@@ -165,40 +153,41 @@ func (self *OFSwitch) IsReady() bool {
 }
 
 // Handle switch connected event
-func (self *OFSwitch) switchConnected() {
-	self.changeStatus(true)
-
-	// Send new feature request
-	self.Send(openflow15.NewFeaturesRequest())
-
-	self.Send(openflow15.NewEchoRequest())
```

Comment:
Is it expected to remove the first echo request?

Reply:
Yes, there is another goroutine to send echo request periodically.